### DISCUSSION TITLE
Add support for powershell like bash ?

### DISCUSSION THREAD

**Discussion by alessandro-celli** [semaphoreui/semaphore#2131] [Ideas]
Hello, 
I would propose to introduce support for PowerShell like bash.
I think that is simple and give a strong enancement to manage windows servers.
A similar solution may be added too for some database like oracle, mssql and mariadb

**Comment by fiftin** [comment]
I think we can add option for Bash Template to select different shells. But currently it is not possible.

**Comment by jengstrom440** [comment]
PowerShell would be most excellent!

**Reply by fiftin** [reply]
Got it, thank you for feedback!
